src/index.js

```diff
@@ -240,32 +255,55 @@ function getConfig(macro, filename, source) {
       const explorer = require('cosmiconfig')('babel-plugin-macros', {
         searchPlaces: [
           'package.json',
-          `.babel-plugin-macrosrc`,
-          `.babel-plugin-macrosrc.json`,
-          `.babel-plugin-macrosrc.yaml`,
-          `.babel-plugin-macrosrc.yml`,
-          `.babel-plugin-macrosrc.js`,
-          `babel-plugin-macros.config.js`,
+          '.babel-plugin-macrosrc',
+          '.babel-plugin-macrosrc.json',
+          '.babel-plugin-macrosrc.yaml',
+          '.babel-plugin-macrosrc.yml',
+          '.babel-plugin-macrosrc.js',
+          'babel-plugin-macros.config.js',
         ],
         packageProp: 'babelMacros',
         sync: true,
       })
       const loaded = explorer.searchSync(filename)
       if (loaded) {
-        return loaded.config[macro.options.configName]
+        configOptions = loaded.config[configName]
+        configPath = loaded.filepath
```

Comment:
Could we break up things a bit into a `getMacroConfigFromConfigFile` and a `getMacroConfigFromBabelConfig`? I fee like having these mixed up makes it more complicated than it needs to be.

Reply:
Sure